lms/product/product.py

```diff
@@ -2,7 +2,7 @@
 
 from dataclasses import InitVar, dataclass
 
-from lms.product.family import Family  # pylint:disable=unused-import
+from lms.product.family import Family
```

Comment:
This import is used below, not sure why we were not getting the pylint alert about it not being needed.